switchbot_ros/msg/Bot.msg

```diff
@@ -0,0 +1,6 @@
+Header  header       # timestamp
+
+float64 battery      # the current battery level, 0-100
+
+string  power        # ON/OFF state
+string  device_mode  # pressMode, switchMode, or customizeMode
```

Comment:
Could you add candidates constants of this field to this message like https://docs.ros.org/en/noetic/api/visualization_msgs/html/msg/Marker.html ?

```
string DEVICEMODE_PRESS = "pressMode"
```

or something like that.

Reply:
I added 3 candedate constants as
```
string DEVICEMODE_PRESS     = "pressMode"
string DEVICEMODE_SWITCH    = "switchMode"
string DEVICEMODE_CUSTOMIZE = "customizeMode"
```